src/main/scala/wdl4s/WdlExceptions.scala

```diff
@@ -0,0 +1,15 @@
+package wdl4s
+
+import scalaz.NonEmptyList
+
+trait ThrowableWithErrors extends Throwable {
+  def message: String
+  def errors: NonEmptyList[String]
+
+  override def getMessage = {
+    s"""$message\n${errors.list.mkString("\n")}"""
+  }
+}
+
+class UnsatisfiedInputsException(message: String) extends RuntimeException(message)
```

Comment:
IIRC Java handles runtime exceptions differently to regular exceptions. But this doesn't feel like it should be one of those special cases - it looks to me like a genuine, predictable, handle-able exception.